packages/react-sdk/src/components/image.tsx

```diff
@@ -1,24 +1,187 @@
 import React, { forwardRef, type ElementRef, type ComponentProps } from "react";
```

Comment:
we need to think where to put the entire logic that is not specific to our image component implementation (if we think this is actually possible)

reason is that someone could write an image component for a different framework, e.g. vue and reuse this logic.

If you agree, then maybe something like src/component-utils ?

Reply:
Moved there with Vercel Licence details